test/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAProjectWithDependencies.cs

```diff
@@ -147,9 +149,11 @@ public void It_publishes_projects_with_simple_dependencies_with_filter_profile()
             });
 
            var runtimeConfig = ReadJson(System.IO.Path.Combine(publishDirectory.ToString(), $"{project}.runtimeconfig.json"));
-
            runtimeConfig["runtimeOptions"]["tfm"].ToString().Should().Be("netcoreapp1.1");
-            
+
+            var depsJson = ReadJson(System.IO.Path.Combine(publishDirectory.ToString(), $"{project}.deps.json"));
+            depsJson["libraries"]["Newtonsoft.Json/9.0.1"]["runtimeStoreManifestName"].ToString().Should().Be($"{manifestFileName1} {manifestFileName2}");
```

Comment:
Looks like you are talking about https://github.com/dotnet/cli/blob/master/test/dotnet-store.Tests/GivenDotnetStoresAndPublishesProjects.cs#L101-L104.

As long as we ensure this test fails when these changes are merged, then I'm fine with continuing to test the end-to-end in the cli.